client/components/ReviewsWidget.jsx

```diff
@@ -44,22 +49,28 @@ export default class ReviewsWidget extends React.Component {
     const { state } = this;
     return (
       <div>
-        <MainHeading>68 reviews</MainHeading>
-        <Stars />
+        <MainHeading>
+          {state.storeCount}
+          {' '}
```

Comment:
I'm not sure what this is for so just making a note saying...'not sure what this is for.'  :)

Reply:
Eslint... this is how you render text + space + props in one line apparently.